roles/nginx/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
Tasks should be in nginx.yml, main.yml should import nginx.yml

Reply:
Import is done